go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
 	sigs.k8s.io/controller-runtime v0.19.0
 	sigs.k8s.io/e2e-framework v0.4.0
+	volcano.sh/apis v1.10.0
```

Comment:
why?

Reply:
Used for volcano gang scheduling e2e test. Deps are light and standard kuberenetes so no hard.